section2/if_else.rb

```diff
@@ -0,0 +1,103 @@
+# Section 2 - If Else
+# Let user pick choice of calculation
+
+def multiply(num_1, num2)
+  num_1 * num2
+end
+
+def divide(num1, num2)
+  num1 / num2
+end
+
+def subtract(num2, num1)
+  num2 - num1
+end
+
+def add(num2, num1)
+  num2 + num1
+end
+
+def modulus(num1, num2)
+  num1 % num2
+end
+
+puts "Simple Calculator"
+  20.times { print "-"}
+puts 
+
+# Declare Variable
+$num_1 
+$num_2
+$userChoice
+
+#1
+loop do
+  puts "Please enter the first number" 
+  $num_1 = gets.chomp
+
+  puts "Please enter the second number"
+  $num_2 = gets.chomp
+
+  puts "What do you want to do?"
+  puts "Enter:"
+  puts "1 to multiply"
+  puts "2 to divide"
+  puts "3 to subtract"
+  puts "4 to add"
+  puts "5 to modulus"
+  $userChoice = gets.chomp
+
+  # To check if the input is Number
+  $num_1 = Float($num_1)
+  $num_2 = Float($num_2)
+
+  # Check userChoice consist of only Integer number
+  $userChoice = Integer($userChoice)
+
+  # Check if userChoice is between 0 to 5
+  # If not, redo the Loop
+  if (!($userChoice >= 0 && $userChoice <= 5))
```

Comment:
1. What if user input 1.1?
2. In your other PR, you insist on comparing float. Means `$userChoice >= 0.0`. Why here you are comparing with integer?

Reply:
>1. What if user input 1.1?
>2. In your other PR, you insist on comparing float. Means $userChoice >= 0.0. Why here you are comparing with integer?

The `$userChoice` is for choosing the type of command user would want to display, for this:
```
  puts "What do you want to do?"
  puts "Enter:"
  puts "1 to multiply"
  puts "2 to divide"
  puts "3 to subtract"
  puts "4 to add"
  puts "5 to modulus"
```
In this case, I'm using Integer to accept the user command from 1 to 5. Therefore, I use `$userChoice = Integer($userChoice)` for this.

If the input is 1.1, it will be thrown to ArgumentError exception & ask user to insert back correct number again.